test_scripts/API/PerformInteraction/011_ATF_PerformInteraction_UI_PerformInteraction_WARNINGS.lua

```diff
@@ -0,0 +1,155 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- SDL must send WARNINGS (success:true) to mobile app in case HMI respond WARNINGS at least to one HMI-portions
+--
+-- Description:
+-- test is intended to check that SDL sends WARNINGS (success:true) to mobile app in case HMI respond WARNINGS to at least one component of RPC
+-- in this test case when UI.PerformInteraction gets WARNINGS and VR.PerformInteraction gets ANY successfull result code is checked
+--
+-- 1. Used preconditions: App is registered and activated SUCCESSFULLY
+-- 2. Performed steps:
+-- MOB -> SDL: sends PerformInteraction
+-- HMI -> SDL: UI.PerformInteraction (WARNINGS), VR.PerformInteraction (cyclically checked cases fo result codes SUCCESS, WARNINGS, WRONG_LANGUAGE, RETRY, SAVED)
+--
+-- Expected result:
+-- SDL -> HMI: resends UI.PerformInteraction and VR.PerformInteraction
+-- SDL -> MOB: PerformInteraction (resultcode: WARNINGS, success: true)
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+-- ToDo (vvvakulenko): remove after issue "ATF does not stop HB timers by closing session and connection" is resolved
+config.defaultProtocolVersion = 2
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require ('user_modules/shared_testcases/commonSteps')
+local testCasesForPolicyTable = require('user_modules/shared_testcases/testCasesForPolicyTable')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+config.SDLStoragePath = commonPreconditions:GetPathToSDL() .. "storage/"
+
+--[[ Local Variables ]]
+local storagePath = config.SDLStoragePath..config.application1.registerAppInterfaceParams.appID.. "_" .. config.deviceMAC.. "/"
+local ServerAddress = commonFunctions:read_parameter_from_smart_device_link_ini("ServerAddress")
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:SDLForceStop()
+commonSteps:DeleteLogsFiles()
+commonSteps:DeletePolicyTable()
+testCasesForPolicyTable:precondition_updatePolicy_AllowFunctionInHmiLeves({"BACKGROUND", "FULL", "LIMITED"},"PerformInteraction")
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+commonSteps:PutFile("Precondition_PutFile", "action.png")
```

Comment:
updated@istoimenova , updated in commit d55b8fe426e072454543d1a1901b705e69b940d9